portfolio/src/main/webapp/script.js

```diff
@@ -12,14 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random fact about myself to the page.
- */
+/* Adds a random fact about myself to the page. */
```

Comment:
Per https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line, JSdoc should be
```
/**
* ...
*/
```
or
```
/** ... */
```